core/pom.xml

```diff
@@ -33,7 +33,7 @@
         <dependency>
             <groupId>org.apache.ant</groupId>
             <artifactId>ant</artifactId>
-            <version>1.9.3</version>
+            <version>1.10.9</version>
```

Comment:
we cannot update to Ant 1.10.x at the moment with this project. See this discussion for more details: https://github.com/UltraMixer/JarBundler/pull/19

Reply:
I merely followed along with dependabot. You clearly have more to deal with than I realised.